scripts/_helpers.py

```diff
@@ -4,13 +4,48 @@
 
 import pypsa
 
-from add_electricity import load_costs, update_transmission_costs
+def configure_logging(logging, snakemake, skip_handlers=False):
```

Comment:
`logging` is a top-level module. There is no reason for it to be passed as an argument.